apps/authorization/handlers.py

```diff
@@ -0,0 +1,53 @@
+from authorization.models import role as role_table
```

Comment:
I think relative imports suit well the new (app-based) project layout:
```python
from .models import role as role_table
from .schemas import AvailableRolesOut
``` 

We can refactor imports later though